integration/ct_integration_test.go

```diff
@@ -99,7 +79,19 @@ func TestInProcessCTIntegration(t *testing.T) {
 	}
 	defer env.Close()
 
-	if err := runParallelTests(cfgs, env.CTAddr, "../testdata", 120*time.Second); err != nil {
-		t.Errorf("%v", err)
-	}
+	mmd := 120 * time.Second
+	// Run a container for the parallel sub-tests, so that we wait until they
+	// all complete before terminating the test environment.
+	t.Run("container", func(t *testing.T) {
```

Comment:
Ah, but there isn't common setup/teardown code -- it's different for different tests.